paper/paper.md

```diff
@@ -0,0 +1,84 @@
+---
+title: 'nbgrader: A Tool for Creating and Grading Assignments in the Jupyter Notebook'
+tags:
+- jupyter
+- jupyterhub
+- grading
+- autograding
+authors:
+- name: Project Jupyter
+- name: Douglas Blank
+  orcid: 0000-0003-3538-8829
+  affiliation: 7
+- name: David Bourgin
+  orcid: 0000-0003-1039-6195
+  affiliation: 1
+- name: Matthias Bussonnier
+  orcid: 
+  affiliation: 1
+- name: Jonathan Frederic
+  orcid: 0000-0003-4805-2216
+  affiliation: 2
+- name: Brian Granger
+  orcid: 
+  affiliation: 3
+- name: Jessica Hamrick
+  orcid: 0000-0002-3860-0429
+  affiliation: 4
+- name: Logan Page
+  orcid: 0000-0002-5799-8524
+  affiliation: 5
+- name: Fernando Pérez
+  orcid: 
+  affiliation: 1
+- name: Benjamin Ragan-Kelley
+  orcid: 0000-0002-1023-7082
+  affiliation: 6
+- name: Carol Willing
+  orcid: 0000-0002-9817-8485
+  affiliation: 3
+affiliations:
+- name: University of California, Berkeley
+  index: 1
+- name: Google Inc.
+  index: 2
+- name: Cal Poly, San Luis Obispo
+  index: 3
+- name: DeepMind
+  index: 4
+- name: University of Pretoria
+  index: 5
+- name: Simula Research Laboratory
+  index: 6
+- name: Bryn Mawr College
+  index: 7
+date: 2 June 2018
+bibliography: paper.bib
+---
+
+*Note: Authors on this paper are listed in alphabetical order.*
+
+# Summary
+
+nbgrader is a tool for creating and grading assignments in the Jupyter notebook [@kluyver2016jupyter]. nbgrader allows instructors to create a single, master copy of the assignment, from which the student version is generated—thus obviating the need to maintain two separate versions. nbgrader automatically grades submitted assignments by executing the notebooks and storing the results in a database. After auto-grading, instructors can provide partial credit or manually grade free-responses using the *formgrader* notebook extension. Finally, instructors can use nbgrader to leave personalized feedback for each student, including instructor comments as well as detailed error information.
```

Comment:
Agree with everything @suchow has noted here. In addition:
- [ ] `nbgrader automatically grades` => `nbgrader also automatically grades` (this is functionality that is in addition to nbgrader's ability to work with a single master copy of an assignment)
- [ ] Last sentence: `instructors can use nbgrader to leave personalized feedback for each student` to `instructors can use nbgrader to leave personalized feedback for each student submission` (making it clear feedback is attached to a student's submission, rather than the student themselves [i.e., if a student doesn't submit anything, they won't receive feedback])